code/modules/jobs/job_types/wasteland.dm

```diff
@@ -289,13 +290,12 @@ Raider
 	suit = /obj/item/clothing/suit/armor/f13/raider/badlands
 	head = /obj/item/clothing/head/helmet/f13/fiend
 	backpack_contents = list(
-		/obj/item/gun/energy/laser/wattz/magneto = 1,
-		/obj/item/gun/energy/laser/wattz = 1,
+		/obj/item/gun/energy/laser/wattz/magneto = 2,
```

Comment:
you cant dual-wield magnetos.

Reply:
raiders as a whole did lose some of their goodies, people should want to be regular bumfuck fiends and have a lore fitting loadout for it, not uber armored commandos like every round.